src/Classes/Entities/StageEntity.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Portal\Entities;
+
+class StageEntity extends ValidationEntity implements \JsonSerializable
+{
+    protected $id;
+    protected $title;
+    protected $order;
+
+    public function __construct(string $id = null, string $title = null, string $order = null)
```

Comment:
docBlock

Reply:
No other entity has a doc block on the constructor